drivers/vulkan/vulkan_context.h

```diff
@@ -184,19 +185,15 @@ class VulkanContext {
 	int command_buffer_count = 1;
 
 	// Extensions.
+	static bool instance_extensions_initialised;
```

Comment:
```suggestion
	static bool instance_extensions_initialized;
```
As much as it hurts, we should use American English here

Reply:
I never get that right :)